drops/azure_edge_extensions_lnm_mq_isa95.json

```diff
@@ -0,0 +1,30 @@
+{
+    "Title": "Azure Edge Extensions AIO ISA-95 LNM and MQ",
+    "Summary": "This is a solution to build an ISA-95 network model on top of Azure IoT Operations utilizing AIO LNM and AIO MQ.",
+    "Description": "This repository defines two levels of an ISA-95 network model and enables MQTT bridging between the layers for bidirectional communication. This sample also showcases how to implement a Kafka Connector to retrieve data from a cloud-based Kafka Broker.",
+    "Cover": "",
+    "Authors": [
+        {
+            "Name": "Nebrass Lamouchi",
+            "Link": "https://github.com/nebrass"
+        },
+        {
+            "Name": "Marcel Bindseil",
+            "Link": "https://github.com/bindsi"
+        }
+    ],
+    "Source": "https://github.com/Azure-Samples/azure-edge-extensions-lnm-mq-isa95",
+    "Type": "sample_app",
+    "Difficulty": "Medium",
+    "ProgrammingLanguage": [
+        "YAML"
+    ],
+    "Products": [
+        "Azure IoT Operations",
+        "Arc",
+        "KafkaConnector"
```

Comment:
Missing space? "Kafka Connector?
What about MQ? ISA-95?

Reply:
agree...KafkaConnector is one single word used in tech...also our AIO docs use it as single word:
[https://learn.microsoft.com/en-us/azure/iot-operations/connect-to-cloud/howto-configure-kafka?tabs=managed-identity#kafkaconnector](https://learn.microsoft.com/en-us/azure/iot-operations/connect-to-cloud/howto-configure-kafka?tabs=managed-identity#kafkaconnector)